spec/controllers/application_controller_spec.rb

```diff
@@ -25,11 +25,17 @@
       @dog = Dog.new("rudolph", "mastiff", 2)
       expect(@dog.age).to eq(2)
     end
-    it 'can change dog age' do
+    it 'can change dog name' do
       @dog = Dog.new("rudolph", "mastiff", 2)
-      @dog. name = "bessie"
+      @dog.name = "bessie"
       expect(@dog.name).to eq("bessie")
     end
+     it 'can change dog age' do
+      @dog = Dog.new("rudolph", "mastiff", 2)
+      @dog.age = "4"
```

Comment:
I think we should be careful to use an Integer here. It doesn't look like there's a db, but just in the name of getting students used to thinking about the types that their using.